src/_includes/config/setup-cron_2.4_how-to.md

```diff
@@ -35,5 +35,10 @@ A sample follows:
 #~ MAGENTO END c5f9e5ed71cceaabc4d4fd9b3e827a2b
 ```
 
-{:.bs-callout-info}
-The `update/cron.php` file exists in [Composer](https://glossary.magento.com/composer)- and [archive-based](https://magento.com/tech-resources/download) Magento installations. It does not exist if you installed Magento by cloning the [Magento 2 git repository](https://github.com/magento/magento2/).
+{%
+include note.html
+type='info'
+content='The update/cron.php has been removed in Magento 2.4.0, if this file exists on your installation, it can be safely removed.
+
+Any reference to update/cron.php should also be removed from the crontab'
+%}
```

Comment:
should add a reference to the setup:cron:run job here as well, which has also been rendered unnecessary and inoperable in version 2.4.0

Reply:
Agreed, both cronjobs were removed in 2.4.0 and this information should probably best be added to the [2.4.0 release notes](https://devdocs.magento.com/guides/v2.4/release-notes/release-notes-2-4-0-open-source.html) as well?

Reply:
Now added